spec/mon_spec.rb

```diff
@@ -24,6 +26,23 @@
       )
     end
   end
+  it do
+    expect(chef_run).to_not create_link('/etc/munin/plugins/cma')
+  end
+  context 'compute node w/ 4.14 kernel' do
+    cached(:chef_run) do
+      ChefSpec::SoloRunner.new(REDHAT_OPTS) do |node|
+        node.set['osl-openstack']['node_type'] = 'compute'
+        node.automatic['kernel']['release'] = '4.14.23-gentoo-osuosl-1.x86_64'
```

Comment:
Is this set to the wrong OS version?

Reply:
Nope, it's using a kernel built on gentoo which prepends this to the kernel version.